diff --git a/node_modules/hast-util-to-html/lib/handle/index.js b/node_modules/hast-util-to-html/lib/handle/index.js
index 31401e1..1dff248 100644
--- a/node_modules/hast-util-to-html/lib/handle/index.js
+++ b/node_modules/hast-util-to-html/lib/handle/index.js
@@ -3,7 +3,8 @@
  * @import {State} from '../index.js'
  */
 
-import {zwitch} from 'zwitch'
+import pkg from 'zwitch';
+const {zwitch} = pkg;
 import {comment} from './comment.js'
 import {doctype} from './doctype.js'
 import {element} from './element.js'
